src/IRrecv.cpp

```diff
@@ -62,11 +62,19 @@ irparams_t *irparams_save;  // A copy of the interrupt state while decoding.
 
 #ifndef UNIT_TEST
 #if defined(ESP8266)
+/// Interrupt handler for when the timer runs out.
```

Comment:
I assume there is no way to avoid this duplication?

Reply:
Not really/not that I've found yet.